README.md

```diff
@@ -1,2 +1,117 @@
 # NASA-RMC-2020-NorthstarRobotics
 The NASA RMC 2020 repository for Northstar Robotics, a segment of University of Minnesota Robotics student group
+
+# Table of Contents
+This is in progress... Will be updated...
+## About
+
+> The goal of the competition is to collect icy regolith simulant from a simulated martian surface and deposit the collected material in a hopper. The top layer of the arena is a fine, dusty material, and the  bottom layer is gravel, which simulates icy regolith on mars. This is accomplished over two runs of 15 minutes. Massive bonus points are available for runs that are fully autonomous, but teleop control is available.
+[Presentation](https://docs.google.com/presentation/d/1eXhP5dbxtgWJBE492b78TFrnQFL9yoPsJFO4HszJn60/edit#slide=id.p)
+
+### Competition Rules
+
+ - [NASA Current
+   Rules](https://www.nasa.gov/offices/education/centers/kennedy/technology/nasarmc.html)
+- [Rulebook 2020](https://www.nasa.gov/sites/default/files/atoms/files/rmc_lunabotics_2020_registration_rules_rubrics_all.pdf)
+- [Our Notes
+   (old)](https://docs.google.com/document/d/1pDDC_1_RaHGiCGMDKFiZ845Ba5-fpn_cems4Xnu7P24/edit)
+
+## Documentation 
+Organization:
+
+ - doc
+ - examples/src
+ - GitSubmoduleUpdateScript.txt
+ - HowToPush
+ - github_pushing
+ - installationHelp.txt
+ - ssh_setup
+
+## Timeline
+
+ - [x] -   Learning
+ - [x] -   Robust Teleop
+ - [ ] -  Trough Alignment
+ 
+ Winter Break
+ - [ ] - Path Planning [FEB]
+	 - [ ]  - Mapping
+	 - [ ] Obstacle Avoidance
+ - [ ]  - Partial Autonomy [MAR]
+	 - [ ] Dumping
+	 - [ ] Driving
+ - [ ] - Full Autonomy [APRIL]
+ - [ ] - Testing [APRIL]
+ - [ ] - Competition [MAY]
+Variable Schedule
+ - [ ] - Simple Driver Assistance
+	 - [ ]  - Auto mining
+	 - [ ] - Lift Leveling
+	 - [ ] - Camera Streaming?
+
+[Milestones](https://docs.google.com/document/d/17NoA1MLxK1FFYn2GWv_bi-EYi50x5UTQvx3rQ464B9A/edit?usp=sharing)
+## ROS Workspace
+The ROS Workspace 
+**Package:**		Description:
+
+* **aruco_localization**  localization using aruco markers (visual targets)	
+* **rovr_common**		  robot specific launch files
+* **rovr_control**	robot control and telecom, wheel io
+* **rovr_description**	a description of the robot, for simulation
+* **rovr_gazebo**		gazebo simulation environment
+* **rovr_input**		testing input/movement
+* **rovr_navigation**	navigation, flexbe robot behaviors
```

Comment:
Do we have pictures of the white board from the other Friday when we explained what went where? 
Did I say the EKF config files go here? 
Btw, this should be a mostly parameter calling of existing ROS path planners. 
Which then begs the necessity of robot_perception [for using cams/lidars to understand the world (very close to _navigation, bounds would have to be defined] and robot_reasoning [less standard, but there really isn't quite a standard yet] for say, probably the state machine, or high-level processing of perceived data (img/pt cloud), such as by keras/tf.